integreat_cms/cms/templates/pages/page_form_sidebar/settings_box.html

```diff
@@ -12,7 +12,19 @@
 {% block collapsible_box_content %}
     <div>
         <label for="{{ page_form.icon.id_for_label }}" class="mt-0">{{ page_form.icon.label }}</label>
-        {% render_field page_form.icon label=page_form.icon.label %}
+        {% if can_edit_page %}
+            {% render_field page_form.icon label=page_form.icon.label %}
+        {% else %}
+            {% if page_form.instance.icon.url %}
+                {# djlint:off H006 #}
+                <img src="{{ page_form.instance.icon.url }}"
+                     class="h-40 m-auto"
+                     alt="{{ page_form.instance.icon.url }}"/>
+                {# djlint:on #}
+            {% else %}
+                {% translate "No icon is set" %}
+            {% endif %}
+        {% endif %}
```

Comment:
```suggestion
        {% render_field page_form.icon label=page_form.icon.label %}
```
Here as well - I'd handle this in the widget template